cmake/SetupDoxygen.cmake

```diff
@@ -34,22 +40,27 @@ if (DOXYGEN_FOUND)
     find_python_module(bs4 FALSE)
     find_python_module(pybtex FALSE)
     if (PY_BS4 AND PY_PYBTEX)
-      # Construct the postprocessing command
+      # Construct the command that runs the postprocessing over the Doxygen HTML
+      # output
       set(
         DOCS_POST_PROCESS_COMMAND
-        ${PYTHON_EXECUTABLE}
-        ${CMAKE_SOURCE_DIR}/docs/config/postprocess_docs.py
-        --html-dir ${PROJECT_BINARY_DIR}/docs/html
-        --references-file ${CMAKE_SOURCE_DIR}/docs/References.bib
+        "${PYTHON_EXECUTABLE} \
```

Comment:
Why does this need changing?

Reply:
Same as above, need to make this a CMake string